File: 2-semestre/uri-online-judge/javascript/1018.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let valor = parseInt(lines.shift());
let atual = valor;

let n100 = parseInt(atual / 100);
atual -= n100 * 100;
let n50 = parseInt(atual / 50);
atual -= n50 * 50;
let n20 = parseInt(atual / 20);
atual -= n20 * 20;
let n10 = parseInt(atual / 10);
atual -= n10 * 10;
let n5 = parseInt(atual / 5);
atual -= n5 * 5;
let n2 = parseInt(atual / 2);
atual -= n2 * 2;
let n1 = parseInt(atual);

console.log(valor);
console.log(n100 + " nota(s) de R$ 100,00");
console.log(n50 + " nota(s) de R$ 50,00");
console.log(n20 + " nota(s) de R$ 20,00");
console.log(n10 + " nota(s) de R$ 10,00");
console.log(n5 + " nota(s) de R$ 5,00");
console.log(n2 + " nota(s) de R$ 2,00");
console.log(n1 + " nota(s) de R$ 1,00");

File: 2-semestre/uri-online-judge/java/1018.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
        int valor;
        double n100, n50, n20, n10, n5, n2, n1;
        
        Scanner entrada = new Scanner(System.in);
        
        valor = entrada.nextInt();
        
        n100 = valor / 100;
        n50 = (valor % 100) / 50;
        n20 = ((valor % 100) % 50) / 20;
        n10 = (((valor % 100) % 50) % 20) / 10;
        n5 = ((((valor % 100) % 50) % 20) % 10) / 5;
        n2 = (((((valor % 100) % 50) % 20) % 10) % 5) / 2;
        n1 = (((((valor % 100) % 50) % 20) % 10) % 5) % 2;
        
        System.out.printf("%d\n", valor);
        System.out.printf("%.0f nota(s) de R$ 100,00\n", n100);
        System.out.printf("%.0f nota(s) de R$ 50,00\n", n50);
        System.out.printf("%.0f nota(s) de R$ 20,00\n", n20);
        System.out.printf("%.0f nota(s) de R$ 10,00\n", n10);
        System.out.printf("%.0f nota(s) de R$ 5,00\n", n5);
        System.out.printf("%.0f nota(s) de R$ 2,00\n", n2);
        System.out.printf("%.0f nota(s) de R$ 1,00\n", n1);
    }
}
